### ISSUE TITLE
Invalid arguments for TmuxWindCmd after vim update to 7.4 patch 1-1584.

### ISSUE THREAD

**Issue by phcerdan** [christoomey/vim-tmux-navigator#110]
The plugin was working with no problems before vim update to 1-1584 in arch.
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Mar 17 2016 10:50:13)
Included patches: 1-1584
Compiled by Arch Linux

Using C-hjkl doesn't work inside vim and generates:
E116: Invalid arguments for function <SNR>79_TmuxWinCmd
Switching panes in tmux works.

**Comment by phcerdan** [comment]
reverting to old version in arch solves* this.
`sudo pacman -U gvim-7.4.1529-1-x86_64.pkg.tar.xz vim-runtime-7.4.1529-1-x86_64.pkg.tar.xz`

**Comment by ManiacalAce** [comment]
I used the [downgrade package](https://aur.archlinux.org/packages/downgrade/)  to downgrade to the above mentioned versions.

`downgrade vim-runtime gvim`

**Comment by akemrir** [comment]
Same here. After downgrade plugin works.

**Comment by saschagrunert** [comment]
Yep same here. Seems vim patch related.

**Comment by nicolaiskogheim** [comment]
It didn't work right after downgrading for some reason, but after restarting Vim a couple of times it worked.

```bash
sudo pacman -U /var/cache/pacman/pkg/vim-7.4.1529-1-x86_64.pkg.tar.xz \
/var/cache/pacman/pkg/vim-runtime-7.4.1529-1-x86_64.pkg.tar.x
```
^ if you don't want to grab the `downgrade` plugin.

**Comment by jck** [comment]
This seems to be fixed in 7.4.1639

**Comment by jeanCarloMachado** [comment]
Solved here by 7.4.1639 as well.

**Comment by djmaze** [comment]
Yes, can confirm it is fixed in 7.4.1639 (on Arch).

**Comment by akemrir** [comment]
I confirm it too. Fixed.

**Comment by christoomey** [comment]
Sounds like this is solved by the most recent patch (7.4.1639) so I'm going to close this. Thanks all for reporting and following up!
